recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
```

Comment:
Nope, that's the new PyPI url scheme.  Old URLs still work, but they get
translated to new ones on their end.  This helps them map exact versions to
unique files.  Issue is at
https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package

On Wed, May 11, 2016 at 4:36 PM Jean-Luc Stevens <notifications@github.com>
wrote:

> In recipes/holoviews-1.5.0/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/588#discussion_r62931378>
> :
>
> > @@ -0,0 +1,41 @@
> > +package:
> > +  name: holoviews
> > +  version: 1.5.0
> > +
> > +source:
> > +  path: .
>
> The current PyPI URL is rather horrible:
>
>
> https://pypi.python.org/packages/96/d2/588631da98d437ee6432999efb6e7f72e626e00f32d28fe154f016f425ec/holoviews-1.5.0.tar.gz
>
> I am hoping this is because the package has just been uploaded and that a
> better URL will be made available soon.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/588/files/72f14d6f05de7bb9519789398f4e3de7e336d98e#r62931378>
>